closure/goog/ui/emoji/emojipicker_test.js

```diff
@@ -810,6 +810,31 @@ testSuite({
     picker.dispose();
   },
 
+  testGetSelectedEmoji_urlPrefix() {
+    const defaultImg = 'http://example.com/../../demos/emoji/none.gif';
+    const picker = new EmojiPicker(defaultImg);
+    picker.setDelayedLoad(false);
+    picker.addEmojiGroup(emojiGroup1[0], emojiGroup1[1]);
+    picker.setUrlPrefix('http://example.com/')
+    picker.render();
+
+    const palette = picker.getPage(0);
+
+    // No emoji should be selected yet
+    assertUndefined(palette.getSelectedEmoji());
+
+    // Artificially select the first emoji
+    palette.setSelectedIndex(0);
+
+    // Now we should get the first emoji back. See emojiGroup1 above.
+    const emoji = palette.getSelectedEmoji();
```

Comment:
This should be the picker object, not the underlying palette, or your changes aren't called.

As the picker's selected emoji is updated via action events (from the palettes it contains), you'll also need to dispatch an ACTION event from the palette to get the picker's getSelectedEmoji to work:

```
// Artificially select the first emoji
palette.setSelectedIndex(0);
palette.dispatchEvent(goog.ui.Component.EventType.ACTION);

// Now we should get the first emoji back. See emojiGroup1 above.
const emoji = picker.getSelectedEmoji();
```